lib/sles4sap.pm

```diff
@@ -135,6 +136,49 @@ undef by default. Test modules testing for SAP ASE should set this property befo
 
 has ASE_RESPONSE_FILE => undef;
 
+=head2 download_hana_assets_from_server
+
+    $self->download_hana_assets_from_server()
+
+Download and extract HANA installation media to /sapinst directory of the SUT.
+The media location must be provided as ASSET_0 in the job settings and be
+available as an uncompressed tar in the factory/other directory of the openQA
+server
+
```

Comment:
Not a request to change just an idea: other library are documenting each argument individually
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/da5a56f4b58f0f322223aad69a7b6d5e2873a521/lib/sles4sap/sap_deployment_automation_framework/deployment.pm#L128